packages/@uppy/provider-views/src/View.js

```diff
@@ -101,7 +101,8 @@ module.exports = class View {
     }
 
     if (file.author) {
-      tagFile.meta.author = file.author
+      if (file.author.name) tagFile.meta.authorName = file.author.name
```

Comment:
Shouldn't we do something like this?
```suggestion
      if (file.author.name != null) tagFile.meta.authorName = String(file.author.name)
```